README.md

```diff
@@ -1,5 +1,13 @@
 # Service Account Management
 
+## Multi-sig process
+
+Flow employs a robust process for executing protocol-level changes, requiring the collective agreement of multiple parties to sign and authorize transactions that trigger key event such as modification of protocol-level parameters. This is commonly referred to as the “multi-sig process”.
+
+Notably, the multi-sig process on Flow is associated with two principal accounts that oversee critical aspects of the Flow protocol: the service account, responsible for actions related to protocol parameters, and the staking account, which holds resources linked to staked nodes within the network. These accounts are collectively overseen by multiple signatories - representatives from the Flow ecosystem - who participate in meetings where transactions are simultaneously authorized (or rejected) and submitted only if there’s quorum. 
+
+At present, representatives from Animoca, Blocto, Flow Foundation, Find, FlowScan, Ichi.org, etc. collectively hold multi-sig authority over the service account. Each signer is allocated a weight of 250 units, and to validate a transaction, a cumulative total of 1000 units is necessary. Therefore, a minimum of four signers must jointly provide their signatures to authorize the transaction. The transactions deliberated by these signatories may be related to minting tokens, setting fees, slashing rewards, updating network-level contracts, etc; see [account page](https://developers.flow.com/build/basics/accounts#service-accounts) for more details on the powers and abilities of the Flow service account.
```

Comment:
Thanks for adding details here. It would be great to add how many active keys of weight `25%` currently exist, since the security and robustness of the "multi-signature" relies on that number. For instance, the guarantees of a (4-out-of-5) are different than (4-out-of-20). 
This info can be added somewhere else, not necessary on this page, especially if it's planned to add a list of the key holders (as in what organization do they represent).

Reply:
@KshitijChaudhary666 - the details Tarak is asking for can be inferred from [flow.json](https://github.com/onflow/service-account/blob/main/flow.json) and [flow-staking.json](https://github.com/onflow/service-account/blob/main/flow-staking.json) if you are planning on adding more details here in the readme

Reply:
Alright, merging this for now but will update this later with more details suggested by Tarak. Thanks guys!